src/shared/pager.c

```diff
@@ -161,8 +171,12 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        safe_fclose(stdout);
-        safe_fclose(stderr);
+        (void) fflush(stdout);
+        (void) dup2(stored_stdout, STDOUT_FILENO);
```

Comment:
hmm, we should probably do this only if stored_stdout is >= 0. We ignore the dupping errors after all when making the copy, hence we should be able to deal with with the copying not having worked.